spec/system/partners/managing_requests_system_spec.rb

```diff
@@ -48,6 +48,17 @@
             last_row.find('option', text: item[:name], exact_text: true).select_option
             last_row.find_all('.form-control').last.fill_in(with: item[:person_count])
           end
+
+          # delete an item
+          find_all('td').last.click
+
+          # BUG: Consider how to make this work. Currently
+          # partners/family_request_create_service validates that no blank values
+          # got passed in. This results in an error. partners/request_create_service filters
+          # all blank items passed in. What is correct behavior?
+          #
+          # Trigger another row but keep it empty. It should still be valid!
+          # click_link 'Add Another Item'
```

Comment:
Not sure if this PR is the place to address this.

Reply:
Agreed, this should be a separate issue/PR.